### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by Godedict**
Portfolio: desktop version project was implemented using the following requirements:

- You need to make sure that your dekstop version looks like the one in the Figma design template that you chose. We include a screenshot of Template 1 below for reference, but you should use the template that you chose.

- IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.

- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px).

- In order to lay out the elements on the page you should use Grid and Flexbox.

- You don't need to implement any functionality that requires JavaScript.

- You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -201,23 +247,16 @@ <h3>Skills</h3><i class="fa-solid fa-angle-right"></i>
...
-          <div class="message">
             <label for="msg"></label>
             <textarea id="msg" name="user_message" maxlength="500" placeholder="Write your message here..."
               required></textarea>
-          </div>
-          <p class="error-message">*invalid email</p>
-          <button class="btn-0">Get in touch</button>
+          <button class="btn-0" type="submit">Get in touch</button>
         </div>
       </form>
```
Comment (Ofarouq310) [inline index.html:261]: - [x] Kindly note that your `Contact` section is missing the background from Figma's design, please, add it.

 | Your design | Figma Design |
| ------------- | ------------- |
|Screenshot 2023-06-10 174159  | image |

File: index.html [hunk 2]
```diff
@@ -74,71 +83,106 @@ <h5>CANOPY</h5>
...
-        <ul class="btn">
-          <li><a href="#" class="html">HTML</a></li>
+        <ul class="btn" id="btn-3">
+          <li><a href="#" class="html" id="html-3">HTML</a></li>
           <li><a href="#" class="css">CSS</a></li>
           <li><a href="#" class="javascript">JavaScript</a></li>
         </ul>
-        <div class="container">
+        <ul class="btn" id="deskbtn-3">
+          <li><a href="#" class="html" id="html-3">html</a></li>
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly note that the element on line `105` contains the same `ID` as the element on line `100`, which conflict with the [`HTML & CSS best practices`](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly fix it.

File: index.html [hunk 3]
```diff
@@ -184,13 +230,13 @@ <h3>Languages</h3><i class="fa-solid fa-angle-down"></i>
             <p>CSS</p>
           </li>
         </ul>
-        <article class="works">
+        <article class="works" id="deskworks">
           <h3>Frameworks</h3><i class="fa-solid fa-angle-right"></i>
         </article>
-        <article class="works">
+        <article class="works" id="deskworks">
```
Comment (Ofarouq310) [inline index.html]: - [ ] Kindly note that the element on line `236` contains the same `ID` as the element on line `233`, which conflict with the [`HTML & CSS best practices`](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly fix it.

File: style.css [hunk 4]
```diff
@@ -648,3 +723,627 @@ section.page-5 {
...
+  .text {
+    margin-top: 3rem;
+    margin-left: 38rem;
+    box-sizing: border-box;
+    width: 45%;
+  }
+
+  .about p {
+    white-space: wrap;
+    padding-top: 1rem;
```
Comment (Ofarouq310) [inline style.css]: [OPTIONAL]
- [ ] Kindly note that `wrap` is not a `white-space` value, thus it does not work, so it is either to select another value or remove the whole property for a cleaner code.

File: style.css [hunk 5]
```diff
@@ -19,12 +21,29 @@ body {
   width: 100%;
   z-index: 1;
   background-color: #fff;
+  position: none;
+}
```
Comment (Ofarouq310) [inline style.css]: [OPTIONAL]
- [ ] Kindly note that `none` is not a `position` value, thus it does not work, so it is either to select another value or remove the whole property for a cleaner code.

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @Godedict, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈 

## Highlights
✅ Descriptive PR title & summary.
✅ Followed GitHub flow.

## Required Changes ♻️

- [x] Kindly note that there is a horizontal bar visible in your project, kindly address which element causes this problem and fix it.

  Screenshot 2023-06-10 174127


- [x] As you can see, the cards in your `Works` section are laid out to the left a bit, according to the design specs, all the cards have to be centered.

 | Your design | Figma Design |
| ------------- | ------------- |
| Screenshot 2023-06-10 174139 | Screenshot 2023-06-10 174018 |


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 5 above)
Comment (Godedict) [reply style.css]: Done thank you

File: style.css (hunk 4 above)
Comment (Godedict) [reply style.css]: changes implemented thanks

File: index.html (hunk 3 above)
Comment (Godedict) [reply index.html]: done, thanks

File: index.html (hunk 2 above)
Comment (Godedict) [reply index.html]: done, thanks

File: index.html (hunk 1 above)
Comment (Godedict) [reply index.html:261]: Changes implemented

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Godedict, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- No Linter errors ✔️
- Good use of Github flow ✔️
- Work on the desktop view as requested by the reviewer 
- Media query properly used

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**